src/Certify.CLI/Program.cs

```diff
@@ -91,6 +94,9 @@ private void PreviewAutoManage()
 
         private bool PerformCertRequestAndIISBinding(string certDomain)
         {
+            // ACME service requires international domain names in ascii mode
+            certDomain = _idnMapping.GetAscii(certDomain);
```

Comment:
this is an entry point where domain name is provided so I made the changes even though I see that this is a work in progress.